app/javascript/app/components/country/country-ndc-sdg-linkages/country-ndc-sdg-linkages-styles.scss

```diff
@@ -17,9 +17,15 @@
   border-left: solid 1px $theme-border;
 
   @include columns(12, $gutters: false);
+  @include xy-gutters($gutter-position: ('bottom'));
+
+  @media #{$tablet-portrait} {
+    @include columns(6, $gutters: false);
+  }
 
   @media #{$tablet-landscape} {
     @include columns(4, $gutters: false);
+    @include xy-gutters($gutter-position: ('bottom'), $gutters: 0);
```

Comment:
This line avoids a separation between the linkages and the footer.  

![image](https://user-images.githubusercontent.com/6906348/37157820-12152dfa-22ea-11e8-8b59-227435f3878c.png)